Gemfile

```diff
@@ -2,17 +2,19 @@
 # specified here will be installed and made available to your morph.io scraper.
 # Find out more: https://morph.io/documentation/ruby
 
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
-ruby "2.0.0"
+ruby '2.3.1'
```

Comment:
I think it probably makes sense to use the latest Ruby version, which is currently [2.3.3](https://www.ruby-lang.org/en/downloads/).